<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/otti2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Syötä kortin numero ja pin koodi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kortti</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>600</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/default.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Kirjaudu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCard">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>label_8</zorder>
     <zorder>label_18</zorder>
     <zorder>loginButton</zorder>
     <zorder>labelInfo</zorder>
     <zorder>label_17</zorder>
     <zorder>lineEditCard</zorder>
     <zorder>lineEditPin</zorder>
    </widget>
    <widget class="QWidget" name="accountPage">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/otti2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>600</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/images/default.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="creditButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Credit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="debitButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Debit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Valitse tili</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
